Update documentation for the Asset Store (#12033)
godotengine/godot-docs: community/asset_store/using_in_editor.rst

.. _doc_using_asset_store_editor:

Using the Asset Store in the Engine
===================================

Accessing it
------------

The Asset Store can be accessed from within Godot from the project manager.

.. image:: img/asset_store_editor_projects.webp

As well as within the editor.

.. image:: img/asset_store_editor_workspace.webp

The Project Manager's :button:`Asset Store` tab will only display assets that are

> standalone projects by themselves. This is denoted on the Asset Store with

Reviewer: ```suggestion
standalone projects. This is denoted on the Asset Store with
```

.. _doc_using_asset_store_editor:

Using the Asset Store in the Engine
===================================

Accessing it
------------

> The Asset Store can be accessed from within Godot at the project manager.

Reviewer: ```suggestion
The Asset Store can be accessed from within Godot from the project manager.
```

.. image:: img/asset_store_editor_projects.webp

As well as within the editor.

.. image:: img/asset_store_editor_workspace.webp

The Project Manager's :button:`Asset Store` tab will only display assets that are

> standalone projects by themselves. This is denoted on the asset library with

Reviewer: ```suggestion
standalone projects. This is denoted on the asset store with
```

the *Template* tag.

The editor's :button:`Asset Store` tab will only display assets that are *not* standalone

> projects by themselves. In other words, it will display assets from all

Reviewer: ```suggestion
projects. In other words, it will display assets from all
```

categories except *Templates*.

If this is the first time you've ever needed the engine to access the internet,

> you'll need to click the :button:`Go Online` Button.

Reviewer: ```suggestion
you'll need to click the :button:`Go Online` button.
```

.. image:: img/go_online.webp

Downloading and installing assets
---------------------------------

> Click on one, and Godot will fetch info about the assets from the Asset Store. Once

Reviewer: ```suggestion
Click on one, and Godot will fetch info about it from the Asset Store. Once
```
I'd say, or even
```suggestion
Click on an asset, and Godot will fetch info about it from the Asset Store. Once
```

it's finished, you will see a window similar to what the Asset Store website looks
like, with some differences:

.. image:: img/asset_store_editor.webp

Similarly to the web version of the Asset Store, here you can search for assets by

> category, name, and sort them by factors such as name or edit date. Unlike when

Reviewer: ```suggestion
category or name, and sort them by name or edit date. Unlike when
```
Or perhaps 

```suggestion
category or name, and sort them by things such as name or edit date. Unlike when
```
I think "factors" is off here

using the web frontend, the search results are updated in real-time (you do not have

> to press Search after every change to your search query for the changes to take

Reviewer: ```suggestion
to press :button:`Search` after every change to your search query for the changes to show).
```
Or
```suggestion
to press :button:`Search` after every change to your search query for the changes to take effect).
```

> place).

Reviewer: ```suggestion
```

When you click on an asset, you will see more information about it.

.. image:: img/asset_store_editor_asset.webp

If you click on the :button:`Download` button, Godot will fetch an archive of the asset,
and will track download progress of it at the bottom of the editor window. If

> the download fails, you can retry it using the Retry button.

Reviewer: ```suggestion
the download fails, you can retry it using the :button:`Retry` button.
```

> When it finishes downloading, the configure asset window will open automatically.

Reviewer: ```suggestion
When it finishes downloading, the Configure Asset window will open automatically.
```
I'd say to make it clear it's a name

.. image:: img/asset_store_editor_configure.png

Here you can see a list of all the files that will be installed. If you click on the

> arrow in the top left, it will open a window where you can tick off any of the files

Reviewer: ```suggestion
arrow on the top left, it will open a window where you can tick off any of the files
```

that you do not wish to install. Any files that can't be installed will be shown in
red, and hovering over them will show you a message stating why it cannot be
installed.

.. image:: img/asset_store_editor_installer_error.png

Once you are done, you can press the :button:`Install` button, which will unzip all the

> files in the archive, and import any assets contained therein, such as images or

Reviewer: ```suggestion
files in the archive, and import any assets contained in it, such as images or
```
I'd suggest

> 3D models. Once this is done, you should see a message stating that the Package

Reviewer: ```suggestion
3D models. Once this is done, you should see a message stating that the package
```
